AndroidP/app/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
YAGNI if you change line 7 (`/build`) in the root gitignore to `build/` — notice the lack of path separator at the beginning, meaning it applies in sub-folders too